homeassistant/components/light/zwave.py

```diff
@@ -52,7 +52,7 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     node = zwave.NETWORK.nodes[discovery_info[zwave.const.ATTR_NODE_ID]]
     value = node.values[discovery_info[zwave.const.ATTR_VALUE_ID]]
     customize = hass.data['zwave_customize']
-    name = super().entity_id
+    name = zwave.ZWaveDeviceEntity(value, DOMAIN).entity_id
```

Comment:
If you make the `_object_id` function name non private in the zwave component, you can use that function here together with the domain name to get the entity_id. Creating an object that you won't use just to get the entity_id seems a bit wasteful.